src/templates/ContentfulPage/index.js

```diff
@@ -7,7 +7,7 @@ export default function ContentfulPageTemplate({ data }) {
   const page = data.contentfulPage
 
   return (
-    <Layout description={page.description} titlePrefix={page.title}>
+    <Layout description={page.description} featureFlags={page.featureFlags} titlePrefix={page.title}>
```

Comment:
Do we need to add this to any other templates?

Reply:
I'm assuming that a ContentfulPage will be the root element for all our content, so I would say no... Are there other root-level types?

Reply:
Yeah, there are for the blog, but you can't set feature flags from WordPress. You should be okay!